### PR TITLE
Feat list

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
Added different list 

- Unordered bullet list 
- Unordered non bullet list 
- Ordered bullet list
- Inline list
- Notification stacked list

Preview: https://deploy-preview-34--hydrogen-ui.netlify.app/docs/list.html

### CODE REVIEW

File: docs/css/docs.css [hunk 1]
```diff
@@ -101,6 +101,10 @@
   margin: 1rem 1rem 1rem 0;
 }
 
+.demo-container :is(ol, ul) {
+  margin-right: 2rem;
+}
+
```
Comment (Hamdrive) [inline docs/css/docs.css]: Use CSS variables if created for rems

**Review by FarhanMobashir — APPROVED** [review]
code is clean. can be merged

**Review by Hamdrive — COMMENTED** [review]
Great work done, components look code and styling implemented are as expected.
Have suggested use of CSS variables, do look into it

File: docs/css/docs.css (hunk 1 above)
Comment (junaidshaikh-js) [reply docs/css/docs.css]: Fixed

**Review by ZuberDunge — APPROVED** [review]

**Review by saiyerniakhil — COMMENTED** [review]
Hey! Good to go. Clean usage of CSS.

**Pull request merged**